core/RELEASENOTES.md

```diff
@@ -0,0 +1,19 @@
+﻿# 2022-02 Release 
+
+# Release Notes
+
+This is the first "formal release" of the IOF Core Ontology that the IOF has made available for public use, and has released as of January of 2023. The focus of the release was primarily to adjust the definitional content of terms to conform with the evolving IOF Annotation Property rules -- the latest version of which is also released herewith. Additionally, the release expands our treatment of measurements and identifiers, and proposes definitional content for terms 'event' and 'state'. Lastly, a key objective was to formalize the infrastructure and processes the IOF uses for managing ontologies in development and during pre- and post-release, and to implement the necessary tooling to automate source code management, testing and deployment.  The new processes, tools and infrastructure have been implemented in this release of Core for the first time and will apply to the all other projects and ontologies the IOF will release in the future. 
+
+##Measurements & Measurement Processes
+Several classes and object properties (relations) have been introduced in this release including measurement process, the ability of a thing to be measured, and the resulting measurement results and measurement value expressions (both informational content entity types).  Many of these are at a “provisional maturity” level and will be refined in the forthcoming release, including better alignment with ISO standards prevailing in this topic area.
```

Comment:
Please change "Several classes" to "Several new classes" and "informational" to "information".

These are the same changes I suggested in email.